my-react/src/CSS/Footer.css

```diff
@@ -0,0 +1,18 @@
+.footer {
+    background-color: #333;
+    color: white;
+    text-align: center;
+    align-items: center;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+  }
+  
+  @media (max-width: 768px) {
```

Comment:
I think we are not implementing this functionality. Please change it accordingly.

Reply:
Implemented the code accordingly.